specification/maps/data-plane/Creator/preview/2022-09-01-preview/routeset.json

```diff
@@ -0,0 +1,360 @@
+        "nextLink": {
+          "description": "If present, the location of the next page of data.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "Routeset": {
+      "description": "Detailed information for the routeset.",
+      "type": "object",
+      "properties": {
+        "routesetId": {
+          "description": "The unique ID of the routeset.",
+          "type": "string",
+          "readOnly": true
+        },
+        "datasetId": {
+          "description": "The ID of the dataset used to create this routeset.",
+          "type": "string",
+          "readOnly": true
+        },
+        "description": {
+          "description": "The description code for the building with the ontology version number.",
+          "type": "string",
+          "readOnly": true
+        },
+        "facilities": {
+          "description": "The facility information and building level ordinal numbers.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/FacilityDetails"
+          },
+          "x-ms-identifiers": [],
+          "readOnly": true
+        },
+        "created": {
+          "description": "The date and time the routeset object was created.",
+          "type": "string",
+          "format": "date-time",
+          "readOnly": true
+        }
+      }
+    }
+  },
+  "parameters": {
+    "ApiVersion": {
+      "name": "api-version",
+      "description": "Version number of Azure Maps API.",
+      "type": "string",
+      "in": "query",
+      "required": true,
+      "default": "2022-09-01-preview",
+      "x-ms-parameter-location": "client"
+    },
+    "RouteDatasetId": {
+      "name": "datasetId",
+      "type": "string",
+      "in": "query",
+      "description": "The ID of the dataset from which to create the routeset(s). A routeset is created for each facility in the dataset.",
+      "required": true,
+      "x-ms-parameter-location": "method"
+    },
+    "Description": {
+      "name": "description",
+      "description": "User provided description of the routeset.",
+      "type": "string",
+      "in": "query",
+      "x-ms-parameter-location": "client"
+    },
+    "QueryRoutesetId": {
+      "name": "routesetId",
+      "description": "The ID of the routeset to query for.",
+      "type": "string",
+      "in": "path",
+      "required": true,
+      "x-ms-parameter-location": "method"
+    },
+    "DeleteRoutesetId": {
+      "name": "routesetId",
+      "description": "The ID of the routeset to delete.",
+      "type": "string",
+      "in": "path",
+      "required": true,
+      "x-ms-parameter-location": "method"
+    },
+    "OperationId": {
+      "name": "operationId",
+      "description": "The ID used to query the status of the routeset creation.",
+      "type": "string",
+      "in": "path",
+      "required": true,
+      "x-ms-parameter-location": "method"
+    }
+  },
+  "paths": {
+    "/routesets": {
+      "post": {
```

Comment:
I think the general recommendation is a PUT. But maybe we already discussed this in meeting, so ignore me if this already settled.

Reply:
yes, we already GA services doing POST, so we do this to stay consistent.